bin/runtime/Cargo.toml

```diff
@@ -109,3 +109,4 @@ std = [
     "pallet-nomination-pools/std",
 ]
 short_session = ["primitives/short_session"]
+treasury_proposals = []
```

Comment:
rename to smth like `enable-treasury-proposals`

Reply:
renamed